docker-run-generate-outputs.sh

```diff
@@ -86,26 +89,40 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
+echo "copying icr files to "$OUTPUT_ICR_DIR" "
 mkdir -p "$OUTPUT_ICR_DIR"
 docker cp "$container:/data/output-icr/." "$OUTPUT_ICR_DIR"
 
+echo "copying auto-coded messages to "$OUTPUT_CODED_DIR" "
 mkdir -p "$OUTPUT_CODED_DIR"
 docker cp "$container:/data/coded/." "$OUTPUT_CODED_DIR"
 
+echo "copying production file to "$OUTPUT_PRODUCTION_CSV" "
 mkdir -p "$(dirname "$OUTPUT_PRODUCTION_CSV")"
 docker cp "$container:/data/output-production.csv" "$OUTPUT_PRODUCTION_CSV"
 
-mkdir -p "$(dirname "$OUTPUT_MESSAGES_CSV")"
-docker cp "$container:/data/output-messages.csv" "$OUTPUT_MESSAGES_CSV"
+if [[ $GENERATE_ANALYSIS_FILES = "true" ]]; then
```

Comment:
This means we're going to have an undescriptive 'true' somewhere in the argument list.

An alternative would be to specify what both modes mean explicitly e.g. $PIPELINE_RUN_MODE = "auto-code-only" | "all-stages" (haven't given much thought to those names). Feel free to open an issue if you don't have time to fix here